code/search/binary_search/README.md

```diff
@@ -9,3 +9,20 @@ The binary search algorithm is used for finding an element in a sorted array. It
 5. The same procedure as in step 2-4 continues, but with a smaller part of the array. This continues until the target element is found or until there are no elements left. 
 
 Collaborative effort by [OpenGenus](https://github.com/opengenus)
+
+// Algo author Yatharth Shah
```

Comment:
The formatting of the following code is not correct and such comments are not allowed in `readme` 👍

Reply:
ok sure will take care sorry! 👍

Reply:
Done the required changes! 👍